src/gsy_e/models/strategy/load_hours.py

```diff
@@ -242,6 +240,10 @@ def __init__(self, avg_power_W, hrs_per_day=None, hrs_of_day=None,
     def state(self) -> LoadState:
         return self._energy_params.state
 
+    @property
+    def asset_type(self):
+        return AssetType.CONSUMER
```

Comment:
I just moved this up closer to the class declaration. 
It's usually good to have the interface (public methods/properties) declared at the beginning of the class, so that the purpose of the class is immediately clear.